test/Sign.Cli.Test/Options/MaxConcurrencyOptionTests.cs

```diff
@@ -33,7 +35,9 @@ public void Option_WhenOptionIsMissing_HasDefaultValue()
         [InlineData(-1)]
         public void Option_WhenValueIsLessThanOne_HasError(int value)
```

Comment:
Is there any limit for number of concurrent connections value? 
Anything more than 64 is not practical, right?

Reply:
Where did you get 64?  An upper bound doesn't appear to be officially documented, so I wouldn't want to hard-code a limit here.

Reply:
I just throwed some random high numbers, maybe not important.